changelog.md

```diff
@@ -17,8 +17,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Programmatically obtain views
 
 ### Added
-- New tables: humanitarian\_needs, funding, refugees, conflict\_events, poverty_rate
-- New enums: Gender, DisabledMarker, EventType, PopulationGroup, PopulationStatus, IPCPhase, PovertyClassification
+- New tables: humanitarian\_needs, funding, refugees, conflict\_event, poverty\_rate, food\_price, wfp\_commodity, wfp\_market, currency
```

Comment:
Can you add these changes to an updated version number instead? As we'll need to tag a new release